docs/dev/config/lookout/config.yaml

```diff
@@ -0,0 +1,44 @@
+grpcPort: 50059
+httpPort: 8089
+metricsPort: 9009
+
+grpc:
+  keepaliveParams:
+    maxConnectionIdle: 5m
+    time: 2h
+    timeout: 20s
+  keepaliveEnforcementPolicy:
+    minTime: 5m
+    permitWithoutStream: false
+
+uiConfig:
+  armadaApiBaseUrl: "http://localhost:8080"
+  userAnnotationPrefix: "armadaproject.io/"
+  binocularsEnabled: true
+  binocularsBaseUrlPattern: "http://localhost:8082" # {CLUSTER_ID} gets replaced by appropriate cluster id
```

Comment:
Same here. Should use binoculars

Reply:
Binoculars doesn't seem to work but unclear if that is due to Fake executor.  I think we can tackle this as a followup since binoculars isn't that important to us at this moment.